doc/source/parameters/unresolved-cfd-dem/void-fraction.rst

```diff
@@ -33,6 +33,7 @@ If the ``mode`` chosen is ``pcm``, then the void fraction is calculated using th
   set quadrature rule              = gauss
   set n quadrature points          = 0
   set project particle velocity    = false
+  set project particle forces      = false
```

Comment:
Is that parameter still there? You moved it to the cfd-dem section, so it should be here no?